templates/payment-fields.php

```diff
@@ -20,7 +20,7 @@
         <div class="securesubmit-content">
             <!-- Start LOGGED IN SAVED CARDS -->
             <?php $cards = get_user_meta(get_current_user_id(), '_secure_submit_card', false); ?>
-            <?php if (is_user_logged_in() && isset($cards)): ?>
+            <?php if (is_user_logged_in() && isset($cards) && !empty($cards)): ?>
```

Comment:
This could be shortened to:
```suggestion
            <?php if (is_user_logged_in() && !empty($cards)): ?>
```
This is because empty() checks also checks whether or not the var is set at all